mob.go

```diff
@@ -1205,11 +1261,78 @@ func getCachedChanges() string {
 
 func makeWipCommit(configuration Configuration) {
 	git("add", "--all")
-	gitWithoutEmptyStrings("commit", "--message", configuration.WipCommitMessage, configuration.gitHooksOption())
+	commitMessage := createWipCommitMessage(configuration)
+	gitWithoutEmptyStrings("commit", "--message", commitMessage, configuration.gitHooksOption())
 	sayInfoIndented(getChangesOfLastCommit())
 	sayInfoIndented(gitCommitHash())
 }
 
+func createWipCommitMessage(configuration Configuration) string {
+	commitMessage := configuration.WipCommitMessage
+
+	lastModifiedFilePath := getPathOfLastModifiedFile()
+	if lastModifiedFilePath != "" {
+		commitMessage += "\nlastFile:" + lastModifiedFilePath
```

Comment:
Shouldn't it be two new lines? Because in the typical git commit message layout, you have the first line, and then an empty line, and then the other info.

Reply:
mh. I can change this, but then I have to change the logic of the squashing again. But I think I should change that anyways.

Reply:
done